contracts/foundry.toml

```diff
@@ -3,4 +3,9 @@ src = "src"
 out = "out"
 libs = ["lib"]
 
+remappings = [
+    'forge-std/=lib/forge-std/src',
+    'coprocessor-base-contract=lib/coprocessor-base-contract/src'
+]
+
```

Comment:
These remappings exist by default and therefore don't need to be specified here.